docs/cloud/private-link.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Private Link
+description: Private Link enables lakeFS Cloud to interact with your infrastructure using private networking.
+parent: lakeFS Cloud
+nav_order: 30
+has_children: false
+redirect_from: "/cloud.html"
+---
+
+# Private-Link
+{: .d-inline-block }
+lakeFS Cloud
+{: .label .label-green }
+
+{: .note}
+> Private Link is only available for [lakeFS Cloud](../cloud/).
```

Comment:
This feels redundant, I think we only need one, WDYT? 

![CleanShot 2023-03-31 at 12 08 54](https://user-images.githubusercontent.com/3671582/229104746-c2886e71-7b7c-41aa-9ad9-6296837fb487.png)